CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 2022.01.1
+
+**Released**: 2022.01.18
+
+**Commit Delta**: [Change from 2021.12.1 release](https://github.com/plus3it/spel/compare/2021.12.1...2022.01.1)
+
+**Manifests**: <https://github.com/plus3it/spel/blob/2022.01.1/manifests>
+
+**Summary**:
+
+*   Publishes image for CentOS 8 Stream
+*   Removes image CentOS 8
+*   "Extra" packages updated in this release:
+    - amazon-ssm-agent-3.1.804.0-1
+    - aws-cli/1.22.37
+    - aws-cli/2.4.11
```

Comment:
This should include:

- aws-cfn-bootstrap (2.0)
- ec2-hibinit-agent-1.0.2-3
- ec2-instance-connect-1.1-15
- ec2-net-utils-1.5-3

And we'll need a bullet on the aws/amazon/ec2 packages that are being removed by default, and provided instead in the `spel` yum repo, enabled by default. So anyone that needs them can use `yum install ...`